submit.py

```diff
@@ -43,6 +43,11 @@ def input(str):
 Metadata = namedtuple("Metadata", ['assignment_key', 'name', 'part_data'])
 Part = namedtuple("Part", ['id', 'input_file', 'solver_file', 'name'])
 
+email_prompt = 'User Name (e-mail address): '
+token_prompt = 'Submission Token (from the assignment page): '
+email_env_var = 'EMAIL_ADDRESS'
```

Comment:
I agree DO_EMAIL would have a a lower chance of "clashing". I was hoping that the clashes would be positive. When I update the pr, I'll change this constant to DO_EMAIL.